From: Serge Semin <fancer.lancer@gmail.com>
Subject: mm/mm_init.c: append newline to the unavailable ranges log-message
Date: Wed, 22 Nov 2023 21:24:04 +0300

Based on the init_unavailable_range() method and it's callee semantics no
multi-line info messages are intended to be printed to the console.  Thus
append the '\n' symbol to the respective info string.

Link: https://lkml.kernel.org/r/20231122182419.30633-7-fancer.lancer@gmail.com
Signed-off-by: Serge Semin <fancer.lancer@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mm_init.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/mm_init.c~mm-mm_initc-append-newline-to-the-unavailable-ranges-log-message
+++ a/mm/mm_init.c
@@ -827,7 +827,7 @@ static void __init init_unavailable_rang
 	}
 
 	if (pgcnt)
-		pr_info("On node %d, zone %s: %lld pages in unavailable ranges",
+		pr_info("On node %d, zone %s: %lld pages in unavailable ranges\n",
 			node, zone_names[zone], pgcnt);
 }
 
_
